docs/fleet/provision.md

```diff
@@ -164,16 +164,54 @@ This provisioning functionality is provided by the [Viam Agent provisioning subs
 
 If you specify a WiFi network to connect to in your configuration file, the Viam Agent will automatically connect to that network when in range.
 
-You can configure one or more WiFi networks to connect to in the `agent` configuration object in your `viam-server` configuration file.
-For example, to configure SSIDs and passwords for two WiFi networks named `primaryNet` and `fallbackNet`, you can use the following configuration:
+To configure the Agent with the necessary WiFi connection information and credentials, add an `agent` configuration object to your `viam-server` configuration file that contains this information.
```

Comment:
Not "neccessary." Additional networks are entirely optional, (and as noted) only typically useful when roaming mode is also enabled.